python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md

```diff
@@ -45,6 +45,9 @@ pip install transformers==4.40
 For optimal performance, it is recommended to set several environment variables. Please check out the suggestions based on your device.
 #### 2.1 Configurations for Windows
 
+> [!NOTE]
+> For optimal performance, we recommend running code in `conhost` rather than Windows Terminal.
```

Comment:
How exactly does one do that? Please provide detailed instructions instead of a simple note.